api/features/serializers.py

```diff
@@ -141,11 +142,20 @@ class FeatureStateSerializerBasic(WritableNestedModelSerializer):
     multivariate_feature_state_values = MultivariateFeatureStateValueSerializer(
         many=True, required=False
     )
+    # TODO: better variable name
+    feature_last_updated_since = serializers.SerializerMethodField()
```

Comment:
@matthewelwell  can you help with the name?

Reply:
Is this a datetime or an integer?

If datetime: `feature_last_updated_at`

If integer: `days_since_feature_last_updated`

Reply:
yeah, it's an integer.

Reply:
Let's go with `days_since_feature_last_updated` then.